metagraph/src/annotation/binary_matrix/row_sparse/row_sparse.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include <sdsl/enc_vector.hpp>
+#include <sdsl/bit_vectors.hpp>
+#include <sdsl/util.hpp>
+#include <sdsl/rrr_vector.hpp>
+
+#include "annotation/binary_matrix/base/binary_matrix.hpp"
+#include "annotation/binary_matrix/multi_brwt/brwt.hpp"
+#include "common/logger.hpp"
+#include "common/vector.hpp"
+#include "graph/annotated_dbg.hpp"
+#include "graph/representation/succinct/boss.hpp"
+#include "graph/representation/succinct/dbg_succinct.hpp"
+
+namespace mtg {
+namespace annot {
+namespace binmat {
+
+class RowSparse : public BinaryMatrix {
+  public:
+    RowSparse() {}
```

Comment:
Could you add it also to all all unit tests (basically to all templated tests with `RowConcatenated` (for RowSparse) or `RowFlatAnnotator` (for RowSparseAnnotator)?